web-app/CKS.Web/Startup.cs

```diff
@@ -50,6 +52,23 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapControllers();
             });
+
+            app.UseStatusCodePagesWithRedirects("/{0}");
```

Comment:
I think we want to return a 404 response, not a 302, so maybe UseStatusCodePagesWithReExecute is probably better, for example something like:

```suggestion
            app.UseStatusCodePagesWithReExecute("/{0}.html");
```